src/Skoruba.IdentityServer4.Admin/DependencyInjection/BuilderExtensions.MultiTenant.cs

```diff
@@ -0,0 +1,78 @@
+﻿using Microsoft.AspNetCore.Identity;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.DependencyInjection;
+using System;
+using Microsoft.AspNetCore.Hosting;
+using Skoruba.IdentityServer4.Admin.EntityFramework.Shared.DbContexts;
+using Skoruba.IdentityServer4.Admin.EntityFramework.Shared.Entities.Identity;
+using IdentityServer4.EntityFramework.Options;
+using Skoruba.IdentityServer4.Admin.BusinessLogic.Identity.Dtos.Identity;
+using Skoruba.IdentityServer4.Admin.EntityFramework.Shared.Stores;
+using Skoruba.IdentityServer4.Admin.EntityFramework.Shared.Managers;
+using System.Linq;
+using Skoruba.IdentityServer4.Admin.EntityFramework.Shared.Validators;
+
+namespace Skoruba.IdentityServer4.Admin.DependencyInjection
+{
+    public static partial class BuilderExtensions
+    {
+        // Default implementation
+        public static Builder AddMultiTenantConfiguration
+            (this IServiceCollection services,
+            IHostingEnvironment hostingEnvironment,
+            Action<DbContextOptionsBuilder> identityDbContextOptions,
+            Action<ConfigurationStoreOptions> configurationStoreOptions,
+            Action<OperationalStoreOptions> operationalStoreOptions,
+            Action<DbContextOptionsBuilder> logDbContextOptions,
+            Action<IdentityOptions> identityOptions)
+        {
+            var builder = new Builder(services);
+            builder.Services.AddMemoryCache();
```

Comment:
probably should be distributed cache, otherwise how to guarantee coherency between this cache and src/Skoruba.IdentityServer4.STS.Identity/DependencyInjection/BuilderExtensions.MultiTenant.cs MemoryCache